packages/nextjs/app/builders/0x21dfd1CfD1d45801f46B0F40Aed056b064045aA2/page.tsx

```diff
@@ -0,0 +1,147 @@
+"use client";
+
+import React from "react";
+import Link from "next/link";
+
+export default function RoheemahBuilderPage() {
```

Comment:
For a Next.js page component, we should follow Next.js conventions, to have a component of type `NextPage`.